src/main/java/hw3/pages/DifferentElementsPage.java

```diff
@@ -0,0 +1,44 @@
+package hw3.pages;
+
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import hw3.pages.components.LogsPanel;
+
+public class DifferentElementsPage extends BasePage {
+
+    private static final String DEFAULT_DIFFERENT_ELEMENTS_URL = "https://jdi-testing.github.io/jdi-light/index.html";
+    private static final String DIFFERENT_ELEMENTS_URL = System.getProperty("url.dif_el", DEFAULT_DIFFERENT_ELEMENTS_URL);
+
+    private final LogsPanel logsPanel;
+
+    public DifferentElementsPage(WebDriver driver) {
+        super(driver);
+        logsPanel = new LogsPanel(driver);
+    }
+
+    @Override
+    public DifferentElementsPage openPage() {
+        driver.get(DIFFERENT_ELEMENTS_URL);
```

Comment:
Cheat: just imagine if base url will change. In how many places you need to update it in that case?


Pls make navigation from main page via clicking on navbar